libs/common/src/platform/services/fido2/fido2-authenticator.service.spec.ts

```diff
@@ -354,6 +366,16 @@ describe("FidoAuthenticatorService", () => {
         cipherService.get.mockImplementation(async (cipherId) =>
           cipherId === cipher.id ? ({ decrypt: () => cipher } as any) : undefined,
         );
+        const cipherData = {} as CipherData;
+        const cipherRecord: Record<CipherId, CipherData> = {
+          [cipherId as CipherId]: cipherData,
+        };
+        const ciphersSubject = new BehaviorSubject<Record<CipherId, CipherData>>(cipherRecord);
+
+        cipherService.ciphers$.mockImplementation((userId: UserId) =>
+          ciphersSubject.asObservable(),
+        );
```

Comment:
suggestion(non-blocking): I think this could be simplified/shortened somewhat 

```suggestion
        cipherService.ciphers$.mockImplementation((userId: UserId) =>
          of({ [cipherId as CipherId]: cipherData })
        );
```

alternatively 

```suggestion
        cipherService.ciphers$.mockImplementation((userId: UserId) =>
          new BehaviorSubject({ [cipherId as CipherId]: cipherData })
        );
```